draft-ietf-sidrops-publication-server-bcp-01.md

```diff
@@ -162,6 +162,72 @@ of the root cause for disruption in the Publication Server that effectively is
 part of their infrastructure, and helps publishers avoid - if possible -
 changes in published RPKI objects that are needed during these windows.
 
+## Availability and Data Loss
+
+The Publication Server MUST ensure high availability of the [@!RFC8181]
```

Comment:
I do not like this requirement. Publishers typically retry and there are no guarantees about how quickly RPKI data propagates (I think multiple hours is typical).

While a repository can not, a publication server can be offline for 15 minutes without problems.

Reply:
We do not specifiy what high availability actually means. Perhaps it would be helpful if we did.

For what it's worth I think the Service Level Objective (SLO) for RRDP and rsync should be around 99.99%, and I agree that RFC8181 can be lower but I believe quite strongly that best practice should be to aim for at least 99.9% here (down 40 minutes per month). Of course higher is always better, but I believe that these values are attainable and good enough.

Publishers indeed retry (Krill every 5 minutes I think). But long outages will make propagation much worse. Multiple hours is not typical - it's the long tail. The medians are much much lower. Furthermore, this is a BCP and BCPs go I strongly feel that they should convey the ambition to do better. Perhaps one day slow validators will become quicker as well - but that is outside of the scope of this document.

Reply:
@ties could you please check if you can live with the updated text in commit https://github.com/timbru/draft-sidrops-publication-server-bcp/pull/30/commits/191a0f0daea987306892f222d6d867b5114eba74